docs/docs/trivy.md

```diff
@@ -10,3 +10,10 @@ The trivy provider is used in Eraser for image scanning and detecting vulnerabil
 * --vuln-type: list of severity levels to report  (default "CRITICAL")
 * --db-repository: repository to use for the trivy vulnerability database (default "ghcr.io/aquasecurity/trivy-db")
 * --delete-scan-failed-images : boolean to delete images for which scanning has failed (default true)
+
+ The following arguments can be used to specify timeouts for the trivy scanner:
+ 
+* --image-scan-timeout: duration for timeout for each of the images scanned (default 1h)
```

Comment:
are we keeping these after #581 merges?

Reply:
I think we can remove it after the configmaps, ill convert this to a draft for now!